CMakeLists.txt

```diff
@@ -4,25 +4,12 @@ project(omim C CXX)
 
 set(CMAKE_CXX_STANDARD 11)
 
-# Set target platform:
-function(omim_set_platform_var PLATFORM_VAR pattern)
-  set(${PLATFORM_VAR} FALSE PARENT_SCOPE)
-
-  if (NOT PLATFORM)
-    if (${ARGN})
-      list(GET ARGN 0 default_case)
-      if (${default_case})
-        set(${PLATFORM_VAR} TRUE PARENT_SCOPE)
-        message("Setting ${PLATFORM_VAR} to true")
-      endif()
-    endif()
-  else()
-    message("Platform: ${PLATFORM}")
-    if (${PLATFORM} MATCHES ${pattern})
-      set(${PLATFORM_VAR} TRUE PARENT_SCOPE)
-    endif()
-  endif()
-endfunction()
+get_filename_component(OMIM_ROOT . ABSOLUTE)
+
+set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${OMIM_ROOT}/cmake")
+
+# Include omim_* functions
+find_package(Omim)
```

Comment:
In general, it looks to me as abuse of find_package, because, actual work is done here, in this CMakeLists.txt. If all what you need is to extract some functions from the main CMakeLists.txt, because it's too large now, can we use plain https://cmake.org/cmake/help/v3.0/command/include.html?

Reply:
Usually find_package is used to import variables from an external module, not for organizing files in the current project.

Reply:
Hm, I was too impressed by find_package after Alohalytics, ```include``` is definitely clearer solution, thanks :)

Reply:
Fixed.